indico/modules/rb_new/util.py

```diff
@@ -541,3 +549,36 @@ def serialize_nonbookable_periods(data):
 
 def serialize_unbookable_hours(data):
     return [bookable_hours_schema.dump(d).data for d in data]
+
+
+def build_rooms_spritesheet():
+    def _resize(image):
```

Comment:
isn't it a bit of overkill to have a function for a single statement used only twice? ;)